src/clj/rems/api/forms.clj

```diff
@@ -37,13 +37,13 @@
   [Form])
 
 (defn- format-form
-  [{:keys [id organization title start endt active? enabled archived]}]
+  [{:keys [id organization title start endt active enabled archived]}]
```

Comment:
There's still endt here?

Reply:
And in all other old code. It's a very invasive refactoring, so better do it in a separate PR. There's a note for it in the ideas backlog.